Core/include/Acts/Clusterization/HkClusterization.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+namespace Acts {
+
+typedef int Label;
+#define NO_LABEL 0
```

Comment:
Can you replace the preprocessor macro with a `constexpr` variable? That way it's namespaced to `Acts`.

Reply:
done